ElasticBox/appliance-vsphere.md

```diff
@@ -10,10 +10,23 @@
 If you don’t have internet access in the vCenter, download all the files and import them locally into your vCenter network. If you have access, you can directly import the appliance in the open virtualization format (OVF) and vCenter will download the disk images for you.
 
 **In this article:**
+<<<<<<< HEAD
```

Comment:
Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.